data_processing/services/radiology_service.py

```diff
@@ -80,6 +91,42 @@
     }
 )
 
+# param models
+collect_parquet_model = api.model("Collect CSVs",
+    {
+        "output_container": fields.String(description="Parquet container qualified address", required=True, example='my_parquet_store'),
+        "input_tags": fields.List(fields.String, description="Input tags to add to container slice", required=True, example=["csv-job-1", "csv-job-2"])
+    }
+)
+
+
+@api.route('/mind/api/v1/<function>/<cohort_id>/submit', methods=['POST'])
```

Comment:
I like the simplicity and generality of this method! It would be great to extend the docstring with some example parameters. In the future, it would be nice to list the functions and add some validations around the params.

Reply:
I think we can make a nice get doc route perhaps?